.github/workflows/ci.yml

```diff
@@ -1,7 +1,12 @@
 name: ci
 
 on:
-  push
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Won't this disable CI if we attempt a PR on another branch? (I don't know if omitting this makes the default all of them or any of them)

Reply:
I think so. Do we ever do that, maybe for intermediate branches?

Reply:
Maybe this is a better config:
```
on: [push, pull_request]
```

Reply:
We have done "PR on top of PR".